utils/config.go

```diff
@@ -0,0 +1,43 @@
+/*
+
+
+Copyright 2020 Red Hat, Inc.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+package utils
```

Comment:
I like moving these out of the controller, but it would be good to make this package more specific. I can foresee `utils` becoming a junk drawer of stuff. Since everything here is related to cloud.redhat, maybe something like `package crhchttp`?

Reply:
Sure. I think that is a true statement in terms of potential junk drawer.

Reply:
rebased and package name updated.